presentation/presentation.go

```diff
@@ -69,13 +69,15 @@ func (c *presentationClient) Share(p params.SharePresentationParams) error {
 		return err
 	}
 
-	err = c.ca.PresentVerifiableCredential(creds, model.Proof{
-		Type:               presentationSig.ProofType,
-		Created:            time.Now().UTC().Format(time.RFC3339),
-		VerificationMethod: presentationSig.VerificationMethod,
-		ProofPurpose:       signer.AssertionMethod.String(),
-		ProofValue:         encoded,
-	}, c.s.GetDid(), p.Challenge.Nonce, signature.ProofValue)
+	vp["proof"] = map[string]interface{}{
```

Comment:
I think it makes sense here to declare a `type  VPProof map[string]interface{}`

Reply:
I think we can investigate that in a follow up, this was just a minimal change for now